worker/src/RTC/SimpleConsumer.cpp

```diff
@@ -248,10 +248,13 @@ namespace RTC
 		return desiredBitrate;
 	}
 
-	void SimpleConsumer::SendRtpPacket(std::shared_ptr<RTC::RtpPacket> packet)
+	void SimpleConsumer::SendRtpPacket(RTC::RtpPacket* packet, std::shared_ptr<RTC::RtpPacket> sharedPacket)
 	{
 		MS_TRACE();
 
+		if (packet == nullptr)
```

Comment:
Wondering if there should be an `MS_ASSERT` here checking whether one of the arguments is not `nullptr`

Reply:
OFC same for every consumer class.

Reply:
Yes